.github/workflows/ci-cd.yml

```
@@ -1,4 +1,5 @@
-name: Tests master
+---
+name: 🧪
```

This could also be
```suggestion
name: 🧪 CI/CD
```
which is short and accurate.

Or
```suggestion
name: CI/CD
```
if you're feeling conservative regarding icons in the UI.

---

I feel conservative (code- and doc-wise, at least).  (I'm old.)

I'd prefer to see the written "CI/CD" in the description so that I don't have to guess as to what the icon means.
Personally, I'd omit the icon altogether — it doesn't add value for me (again, I'm old) — though I don't feel strongly about that.

---

Let's try having both the icon and text, then. Here's an example of what it looks like when jobs are also labeled similarly: https://github.com/ansible/awx-plugins/actions/runs/11707588320.